docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single-node.md

```diff
@@ -70,7 +70,7 @@ In the example below the player who placed x has won.
 When all the squares contain a marker and no player has three markers in a row, then the game is a draw.
 
 ### Accounts and Key Pairs
-Accounts are stored on the blockchain with a public key. Use a private key to access the account, though be sure to secure your private key. For more information about account and permissions click on this link [Accounts and Permissions.](../04_protocol/04_accounts_and_permissions.md "Accounts and Permissions Overview")
+A blockchain account has a human readable name which is between 1 and 12 characters in length. Each account identifies a blockchain participant and the authority of that participant. Loading a smart contracts requires an account; an account can own one smart contract instance and a smart contract instance must be loaded by an account. Accounts are stored on the blockchain with their public keys. Each account requires at least one key pair (public and private keys.) The blockchain uses asymmetric cryptography to verify that the account pushing a transaction has signed the transaction with the matching private key. EOSIO blockchains use account authority tables to check that the account has the required authority to perform an action. For more information about accounts and permissions click on this link [Accounts and Permissions.](../04_protocol/04_accounts_and_permissions.md "Accounts and Permissions Overview")
```

Comment:
Instead of `loading ` a smart contract I would use `to upload a smart contract to the blockchain`.
why? first it avoids the gerund (`ing`).
and second, I personally did not encounter this terminology of `loading a smart contract` much, however I did find it a lot more used the other one.

Reply:
speaking about gerunds :) (`ing`s)
I would search in the whole document for them and try to correct them.
or create a task that is all about this correction.